CoreScriptsRoot/Modules/Server/ClientChat/ObjectPool.lua

```diff
@@ -0,0 +1,54 @@
+--	// FileName: ObjectPool.lua
+--	// Written by: TheGamer101
+--	// Description: An object pool class used to avoid unnecessarily instantiating Instances.
+
```

Comment:
The developer console doesn't have a module, it just keeps labels in a table and reuses them like that. I remember there being a different version of the developer console that wasn't all one script that I think did it better but that isn't the current one.